langserver/config.go

```diff
@@ -105,6 +110,9 @@ func (c *Config) Validate() error {
 		// default value
 		c.LogFormat = TextFormat
 	}
+	if c.Interval <= 0 {
```

Comment:
This should be logged.

Reply:
with which logger ?

Reply:
Good question. Ideally we'd get that log message in into the lsp client. But that'd require some re factoring.

Reply:
Maybe let's skip this for now.